clash-lib/prims/systemverilog/Clash_Signal_Internal.primitives.yaml

```
@@ -231,4 +231,5 @@
       `endif
       // pragma translate_on
       // resetGen end
+    warning: Clash.Signal.Internal.resetGenN is not synthesizable!
```

```suggestion
    warning: Clash.Signal.Internal.resetGenN is not synthesizable to hardware!
```

To keep it consistent

---

That's a weird sentence fragment, though. It's, if my grammar does not escape me, an adverbial phrase to an adjective. That in itself is not wrong but it feels awkward in this form. What about

```suggestion
    warning: Clash.Signal.Internal.resetGenN can not be synthesized to hardware!
```